pkg/equinixmetal/utils.go

```diff
@@ -29,6 +29,7 @@ func findSelf(c *packngo.Client, projectID string) *packngo.Device {
 	// Go through devices
 	dev, _, _ := c.Devices.List(projectID, &packngo.ListOptions{})
 	for _, d := range dev {
+		// TODO do we need to replace os.Hostname with config.NodeName here?
 		me, _ := os.Hostname()
```

Comment:
Maybe here we also want to use node name as reported by k8s, but since I don't have BGP setup to test it, I left it as is, to avoid breaking stuff.